app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Kindly ensure that the foreign key is referenced correctly in the comments and likes, much like what you did with the posts: `foreign_key: 'author_id'`.
- [ ] Please create a method for the user model that returns the 3 most recent posts for a given user, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=A%20method%20that%20returns%20the%203%20most%20recent%20posts%20for%20a%20given%20user.) requirement.